### DISCUSSION TITLE
build issue >:((((

### DISCUSSION THREAD

**Discussion by NicKThEStiCk1230** [KadeArchive/Kade-Engine#1784] [Q&A]
so uhh whenever i try to compile using "lime build windows" it always comes up with 
"Standard library not found. You may need to set your `HAXE_STD_PATH` environment variable"

so like uh

**help me**

**Comment by alphox02** [comment]
idk, maybe try putting this on issues and someone will notice it

**Comment by BoloVEVO** [comment]
Try reinstalling the HaxeToolKit. Instructions are in building KE page.
